File: uphold-integration/src/main/java/org/dash/wallet/integration/uphold/ui/UpholdDisabledActivity.java
package org.dash.wallet.integration.uphold.ui;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import org.dash.wallet.common.InteractionAwareActivity;
import org.dash.wallet.integration.uphold.R;

public class UpholdDisabledActivity extends  InteractionAwareActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.uphold_disabled_screen);
        findViewById(R.id.go_to_uphold).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String url = "https://uphold.com";
                Intent i = new Intent(Intent.ACTION_VIEW);
                i.setData(Uri.parse(url));
                startActivity(i);
            }
        });

    }
}

File: uphold-integration/src/main/java/org/dash/wallet/integration/uphold/ui/UpholdDisabledActivity.kt
/*
 * Copyright 2020 Dash Core Group
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

package org.dash.wallet.integration.uphold.ui

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import kotlinx.android.synthetic.main.uphold_disabled_screen.*
import org.dash.wallet.common.InteractionAwareActivity
import org.dash.wallet.integration.uphold.R

class UpholdDisabledActivity : InteractionAwareActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.uphold_disabled_screen)
        cancel_button.setOnClickListener {
            finish()
        }
        go_to_uphold.setOnClickListener {
            val url = "https://uphold.com"
            val i = Intent(Intent.ACTION_VIEW)
            i.data = Uri.parse(url)
            startActivity(i)
        }
    }

}